manim/mobject/text/text_mobject.py

```diff
@@ -1331,7 +1331,8 @@ def _parse_color(self, col):
         if re.match("#[0-9a-f]{6}", col):
             return col
         else:
-            return Colors[col.lower()].value
+            testCol = col.upper()
+            return globals()[testCol] 
```

Comment:
I'm not sure if using globals() is good practice? We should instead store all the colors in some data structure and use that one. As suggested by @Darylgolden:

> Regarding the first point: I've realized that [random_color](https://github.com/ManimCommunity/manim/blob/7efa2066acfe924abf30af2965167670843ed6c8/manim/utils/color.py#L537-L538) relies on an object containing information on all preset colors, so perhaps it might not be desirable to remove such an object entirely. Perhaps Colors should be a list? I'm not sure.

Reply:
Good point! I've changed how the values can be accessed now